Code/Core/Math/Random.cpp

```diff
@@ -24,4 +26,15 @@ uint32_t Random::GetRand()
     return( (uint32_t)( m_Seed / ( ( CORE_RAND_MAX + 1 ) * 2 ) ) % ( CORE_RAND_MAX + 1 ) );
 }
 
+// GetRandString
+//------------------------------------------------------------------------------
+uint32_t Random::GetRandString( AString& str )
+{
+    uint32_t rand_num = GetRand();
```

Comment:
FASTBuild's coding style uses camel case, so this variable would usually be called "randNum".

This variable could also be const.

I can of course fix minor issues like this, but the fewer there are the easier things will be.

Reply:
I removed this entire function following your earlier suggestion.